integration-tests/terraform/eks/deployment/variables.tf

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "k8s_version" {
```

Comment:
Mind creating a follow up task to create a matrix and run this integ test for diff k8s versions.

Reply:
Will do